addons/ekco/template/base/install.sh

```diff
@@ -54,6 +54,11 @@ function ekco_pre_init() {
         LOAD_BALANCER_ADDRESS=localhost
         LOAD_BALANCER_PORT="6444"
     fi
+
+    EKCO_MAINTAIN_MINIO=false
+    if [ -n "$MINIO_VERSION" ] && [ -n "$OPENEBS_LOCALPV" ]; then
+        EKCO_MAINTAIN_MINIO=true
```

Comment:
if you add minio and dont change the ekco version this will not get applied. should we always run ekco_create_deployment in ekco_already_applied?

Reply:
Yeah, that seems like a good idea. That might be meat for a different PR though